lib/pages/portal/view/website_view.dart

```diff
@@ -55,9 +54,10 @@ class _WebsiteViewState extends State<WebsiteView> {
 
   User user;
 
-  Uint8List uploadedImageBytes;
   ImageProvider imageWidget;
   TextEditingController imageFieldController = TextEditingController();
+  UploadButtonController uploadButtonController;
+  UploadButton uploadButton;
```

Comment:
Having a variable where you're storing an already-built widget is not necessary in 99% of cases, and can lead to issues like widgets not updating properly.

Remove this variable, you can access the controller by using the uploadButtonController variable.